packages/docs/src/routes/docs/(qwik)/components/overview/index.mdx

```diff
@@ -286,7 +286,7 @@ You can think of inline components as being inlined into the component where the
 ### Limitations
 Inline components come with some limitations that the standard `component$()`
 does not have. Inline components:
-- Cannot use `use` methods such as `useSignal` or `useStore`.
+- Cannot `use` methods such as `useSignal` or `useStore`.
```

Comment:
imo the word `use` is correct. the 2nd `use` stands for all hooks like `useVisibleTask`, `useOn`. guess the best update here would be smth like this:

```suggestion
- Cannot use `use*` methods such as `useSignal` or `useStore`.
```